cmd/device-hub/cmd_server.go

```diff
@@ -29,6 +30,12 @@ var serverCommand = &cobra.Command{
 
 		var dataImpl store.Storer
 
+		// Load Config File if is enabled
+		if _config.ConfigFile {
+			log.Println("Overriding all settings with config file")
+			_config.AddConfigFile(_config.ConfigPath)
+		}
+
```

Comment:
Could we move this logic into the main.go init function? My reason would be that any additional cmds would also need this stanza added.